man/docker-run.1.md

```diff
@@ -52,6 +52,7 @@ docker-run - Run a command in a new container
 [**--restart**[=*RESTART*]]
 [**--rm**[=*false*]]
 [**--security-opt**[=*[]*]]
+[**-s**|**--stop-signal**[=*SIGNAL*]]
```

Comment:
also wondering if we need a short-hand (`-s`) flag for this; will this be a frequently enough used option? (we're running out of letters fast :-))